compiler/include/resolution.h

```diff
@@ -146,7 +146,8 @@ void wrapAndCleanUpActuals(ResolutionCandidate* best, CallInfo& info,
 typedef enum {
   FIND_EITHER = 0,
   FIND_REF,
-  FIND_NOT_REF
+  FIND_CONST_REF,
+  FIND_NOT_REF_OR_CONST_REF,
```

Comment:
My brain doesn't parse this well. Is it ``!ref || const_ref`` or ``!(ref || const_ref)`` ? Would an identifier like ``FIND_VALUE`` be better, if that's what it's supposed to do?